test/storage/storage_test.cpp

```diff
@@ -0,0 +1,266 @@
+#include <gtest/gtest.h>
+
+#include "veriblock/arith_uint256.hpp"
+#include "veriblock/storage/block_repository_inmem.hpp"
+#include "veriblock/storage/block_repository_rocks_setup.hpp"
+#include "veriblock/uint.hpp"
+
+using namespace VeriBlock;
+
+enum class TEST_RUNS { INMEM = 0, ROCKS_BTC, ROCKS_VBK };
+
+struct BlockBasic {
+  using hash_t = uint256;
```

Comment:
why did you change simple-to-use `int` to hard-to-use `hash256`?

Reply:
Because RocksDB assumes keys as byte arrays. Integers cannot be accessed the same way - they do not have neither data() nor size().

Reply:
I updated the code to use getHash() from the block interface and convert int to hash_t for BasicBlock. Code is cleaner now.